pwa/src/templates/componentDetail/ComponentsDetailTemplate.tsx

```diff
@@ -343,78 +343,27 @@ export const ComponentsDetailTemplate: React.FC<ComponentsDetailTemplateProps> =
-                      <TableCell>
-                        <Link icon={<GitHubLogo />} iconAlign="start">
-                          Componenten GitHub
-                        </Link>
-                      </TableCell>
-                      <TableCell>
-                        <Link icon={<CallIcon />} iconAlign="start">
-                          010 - 123 456 7
-                        </Link>
-                      </TableCell>
-
-                      <TableCell
-                        className={styles.details}
-                        onClick={() => navigate("/organizations/5b9e0b17-00ca-433c-961b-913270643e6d")}
-                      >
-                        <Link icon={<ArrowRightIcon />} iconAlign="start">
-                          {t("Details")}
-                        </Link>
-                      </TableCell>
-                    </TableRow>
-
-                    <TableRow>
-                      <TableHeader>Conduction</TableHeader>
-                      <TableCell>
-                        <Link icon={<GitHubLogo />} iconAlign="start">
-                          Componenten GitHub
-                        </Link>
-                      </TableCell>
-                      <TableCell>
-                        <Link icon={<CallIcon />} iconAlign="start">
-                          020 - 123 456 7
-                        </Link>
-                      </TableCell>
-
-                      <TableCell
-                        className={styles.details}
-                        onClick={() => navigate("/organizations/5b9e0b17-00ca-433c-961b-913270643e6d")}
-                      >
-                        <Link icon={<ArrowRightIcon />} iconAlign="start">
-                          {t("Details")}
-                        </Link>
-                      </TableCell>
-                    </TableRow>
-
-                    <TableRow>
-                      <TableHeader>Gemeente Utrecht</TableHeader>
-                      <TableCell>
-                        <Link icon={<GitHubLogo />} iconAlign="start">
-                          Componenten GitHub
-                        </Link>
-                      </TableCell>
-                      <TableCell>
-                        <Link icon={<CallIcon />} iconAlign="start">
-                          030 - 123 456 7
-                        </Link>
-                      </TableCell>
-
-                      <TableCell
-                        className={styles.details}
-                        onClick={() => navigate("/organizations/5b9e0b17-00ca-433c-961b-913270643e6d")}
-                      >
-                        <Link icon={<ArrowRightIcon />} iconAlign="start">
-                          {t("Details")}
-                        </Link>
-                      </TableCell>
-                    </TableRow>
-                  </TableBody>
-                </Table>
+                {tempOrganization.map((organization) => (
```

Comment:
I think these organisations are based on `maintenance.contractors`, which are available in the testdata.

Component "Ultricies Nec" holds one contractor (other might as well).

Please (1) check if `maintenance.contractors` indeed is the right place to look for leveranciers and (2) implement the real data.

(Probably should validate it with @rubenvdlinde)

Reply:
If there's no organisations available, we want to render a message saying so, like so:

```
!arrayOfContractors.length && <>Dit component heeft geen leveranciers.</>
```

Reply:
Neuh levernciers kan je bepalen aan de hand van de leveranciers pagina

Reply:
Communication in Slack:

Ik ben bezig met een pull request die organisation cards aan componenten toe moet voegen. Hier stond nog [deze comment](https://github.com/OpenCatalogi/web-app/pull/134/files#r1004215377) open van jou open.
Aantal vragen hierover / opmerkingen:
Ik zie dat de organisation objecten een supports array hebben, in de testdata is deze (soms) gevuld met een link (e.g. https://github.com/OpenCatalogi/web-app)
In de PublicCode staat dat dit een array aan uuids zou moeten zijn, dat komt nu niet overeen met wat ik in de response krijg. Moet het de url blijven of moeten dit uuids worden?
Als het de url moet blijven, waar baseren we dit dan op vanuit het component? (e.g. isBasedOn of landingURL lijken overeen te komen)
Wanneer ik een request doe met de URL (e.g. .../oc/organizations?supports[]=https://github.com/OpenCatalogi/web-app, dan krijg ik een gigantische set terug.
Er bestaan 1135 bedrijven (allemaal "ConductionNL" die dit component ondersteunen)
Er bestaan tevens 1197 bedrijven die "ConductionNL" heten (zie punt 2.a.)

Reply:
@rubenvdlinde ^